### ISSUE TITLE
Grin

### ISSUE THREAD

**Issue by xaur** [RichardRed0x/crypto-governance-research#6]
While briefly reading for the Journal I noticed a few interesting aspects of Grin:

1) people request funding on individual basis, with transparent amounts and open discussion https://www.grin-forum.org/t/request-for-funding-lehnberg/4715
2) decisions are made on governance meetings (I guess mostly chat?) with meeting notes posted on github https://github.com/mimblewimble/grin-pm/tree/master/notes
3) decisions are reported in a public decision log https://github.com/mimblewimble/grin-pm/blob/master/decision_log.md
4) financials are also reported https://github.com/mimblewimble/grin-pm/tree/master/financials

Also interesting is Grin's research on foundations https://github.com/mimblewimble/docs/wiki/Regarding-Foundations

I love how these guys make effort to post everything on github.

**Comment by s-ben** [comment]
Interesting...Saw this CoinDesk article on them [hiring](https://www.coindesk.com/privacy-cryptocurrency-grin-votes-to-fund-third-full-time-developer) a 3rd full-time dev.  I wonder how the decision-making will scale with the project (assuming it does). From the article:

> “There’s no [official] roles. We just continue to work on what we see fit, which so far seems to be working okay.” - Yeastplume , 1st full-time paid developer

The "roughy $10k/mo USD shows up as $25k for 3 months in their spending report. Presumably they do some kind of averaging when paying out in Grin.

**Comment by xaur** [comment]
Yep an the comment column (scroll to the right) says "not enough grins for full 30k payment so rest to be completed when possible".

**Comment by s-ben** [comment]
$8.3k/mo still ain't bad...I wonder how much they're [getting](https://blog.circle.com/2019/02/14/poloniex-welcomes-grin/) from Poloniex transaction fees. Seems to be their only source of funding outside donations (which I don't think is viable long-term unless you count VC money funding your devs as "donations").

**Comment by xaur** [comment]
First donation [was](https://blog.circle.com/2019/03/15/poloniex-enables-grin-withdrawals-donates-to-grin-general-fund/) 1.3 BTC. I think donations is a viable but unpredictable and unstable source. It may work if humans are properly organized and understand the importance of funding the chair they are sitting on.

**Comment by s-ben** [comment]
> and understand the importance of funding the chair they are sitting on.

Ha! Saw this recently researching tip bots. Recurring [micropayments](https://ethereumworldnews.com/progress-xrptipbot-now-has-a-recurring-feature/) in the XRP tip bot (a huge growth engine for them). Interesting idea. Depressing, as it's just more digital meth for the XRP Army. But interesting.
